components/Map/Map.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import districtJson from '../../geoJSON/district.json';
+import subcountyJson from '../../geoJSON/subcounty.json';
+import * as turf from '@turf/turf';
+import * as distance from 'jaro-winkler';
+
+const style = {
+  width: '100%',
+  height: '600px'
+};
+
+interface MapProps {
+  district: string;
+  boundaryType: string;
+}
+
+interface State {
+  leaflet: {};
+  map: {};
+}
+
+class Map extends React.Component<MapProps, State> {
+  constructor(props: MapProps) {
+    super(props);
+    this.state = {
+      leaflet: {},
+      map: {}
+    };
+  }
+
+  componentDidMount() {
+    const L = require('leaflet');
```

Comment:
Nope ... imports at the top, unless it's a dynamic import.

Reply:
Now using dynamic imports and this import works at the top